### ISSUE TITLE
"New Module Imports" rule does not flag Ember.Test destructuring

### ISSUE THREAD

**Issue by hakilebara** [ember-cli/eslint-plugin-ember#217]
This example does not trigger an eslint error:
```javascript
import Ember from 'ember';

const {
  Test
} = Ember;

export default Test.registerAsyncHelper('setBreakpoint', function(app, breakpoint) {
 ...
});
```
I believe the new-module-imports rule should recommend me to use  `import { registerAsyncHelper } from '@ember/test';` instead

**Comment by rwjblue** [comment]
Agreed! Thanks for reporting!

**Comment by jbandura** [comment]
I've investigated the issue and the main problem seems to be that we don't check for cases like:
```js
const { Test } = Ember;
```
or 
```js
const { String } = Ember;
```
If the above snippet would be changed to:
```js
import Ember from 'ember';

const {
  Test: { registerAsyncHelper },
} = Ember;

export default registerAsyncHelper('setBreakpoint', function(app, breakpoint) {
 ...
});
```
it would trigger the warning. I can try and have a shot at implementing warnings for such cases :)
